lib/postgrex.ex

```diff
@@ -118,11 +117,10 @@ defmodule Postgrex do
     * `:idle_interval` - Ping connections after a period of inactivity in milliseconds.
       Defaults to 1000ms;
 
-    * `:ssl` - Set to `true` if ssl should be used (default: `false`);
-
-    * `:ssl_opts` - A list of ssl options, see the
-      [`tls_client_option`](http://erlang.org/doc/man/ssl.html#type-tls_client_option)
-      from the ssl docs;
+    * `:ssl` - Enables SSL. Setting it to `true` enables SSL without host verification,
+      which emits a warning. Instead, prefer to set it to a keyword list, with either
+      `:cacerts` or `:cacertfile` pointing to the server certificate, to enable hostname
+      verification. Defaults to `false`;
```

Comment:
Setting `verify_peer` enables server certificate verification, with hostname verification being just a part of that. Without it, the server certificate is completely ignored, not just the hostname.

Also, `:cacerts` and `:cacertfile` need to point to a CA trust store, not to the server certificate.

```suggestion
    * `:ssl` - Enables SSL. Setting it to `true` enables SSL without server certificate verification,
      which emits a warning. Instead, prefer to set it to a keyword list, with either
      `:cacerts` or `:cacertfile` set to a CA trust store, to enable server certificate
      verification. Defaults to `false`;
```